examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
It does, but you have to guess 300, a guess value of 200 is not good enough. See also #120, where scipy's curve fit can fit with 200 as initial guess. Contrary to the finding in that issue, if I switch to ``ConstrainedNumericalLeastSqaures`` here, it doesnt solve the "issue", fitting still sometimes (depending on seed) fails with guess 200. I'll have another look at #120 if I have time later.